backend/src/test/java/com/google/sps/FindMatchQueryTest.java

```diff
@@ -0,0 +1,184 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.sps.data.Match;
+import com.google.sps.data.Participant;
+import com.google.sps.data.TimeHelper;
+import java.time.Clock;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class FindMatchQueryTest {
+
+  // Some people that we can use in our tests
+  private static final String PERSON_A = "Person A";
+  private static final String PERSON_B = "Person B";
+  private static final String PERSON_C = "Person C";
+  private static final String PERSON_D = "Person D";
+
+  // Default parameters unused in query
+  private static final long ID_DEFAULT = 0;
+  private static final long TIMESTAMP_DEFAULT = 0;
+
+  // Some times available until on 1/1/2020
+  private static final ZonedDateTime TIME_0200PM = TimeHelper.getNewTimeToday(14, 0);
```

Comment:
since TimeHelper's constructor isn't called in this test, shouldn't this cause a null pointer exception? That's another reason to make getNewTimeToday an instance method.